projects/plugins/protect/src/js/components/admin-page/index.jsx

```diff
@@ -46,11 +47,39 @@ const InterstitialPage = ( { run, hasCheckoutStarted } ) => {
 };
 
 const ProtectAdminPage = () => {
-	const { lastChecked } = useProtectData();
-
+	const { lastChecked, currentStatus, errorCode, errorMessage } = useProtectData();
 	// Track view for Protect admin page.
 	useAnalyticsTracks( { pageViewEventName: 'protect_admin' } );
 
+	// Error
+	if ( 'error' === currentStatus ) {
+		let displayErrorMessage = errorMessage
+			? `${ errorMessage } (${ errorCode }).`
+			: __( 'We are having problems scanning your site.', 'jetpack-protect' );
+		displayErrorMessage += ' ' + __( 'Try again in a few minutes.', 'jetpack-protect' );
+
+		return (
+			<AdminPage moduleName={ __( 'Jetpack Protect', 'jetpack-protect' ) } header={ <Logo /> }>
+				<AdminSectionHero>
+					<Container horizontalSpacing={ 3 } horizontalGap={ 7 }>
+						<Col sm={ 4 } md={ 4 } lg={ 6 }>
+							ALERT ICON GOES HERE
+							<H3 mt={ 8 }>
+								{ __( 'We’re having problems scanning your site', 'jetpack-protect' ) }
+							</H3>
+							<Text>{ displayErrorMessage }</Text>
+						</Col>
+						<Col sm={ 0 } md={ 0 } lg={ 1 }></Col>
+						<Col sm={ 4 } md={ 3 } lg={ 5 }>
+							<img src={ inProgressImage } alt="" />
+						</Col>
+					</Container>
+				</AdminSectionHero>
+				<Footer />
+			</AdminPage>
```

Comment:
let's try to avoid this composition. Take a look at the [Dialog component](https://github.com/Automattic/jetpack/blob/master/projects/js-packages/components/components/dialog/index.jsx) instead.
Special mention for `<Col sm={ 0 } md={ 0 } lg={ 1 }></Col>` :-D